Sming/third-party/.patches/axtls-8266.patch

```diff
@@ -114,13 +114,10 @@ diff -Nuar a/replacements/mem.c b/replacements/mem.c
 +
 +void ax_port_free(void* ptr) {
 +    free(ptr);
-+    uint32_t *p = (uint32_t*) ptr;
-+    size_t size = p[-3];
-+    if (size >= 1024)
-+        DEBUG_TLS_MEM_PRINT("free %d, left %d\r\n", p[-3], system_get_free_heap_size());
++    DEBUG_TLS_MEM_PRINT("free %d, left %d\r\n", p[-3], system_get_free_heap_size());
```

Comment:
This code was used to try and guess the length of the freed chunk. If freed chunlk was over 1024 => printf.
This is however not portable, when using for ex umm_malloc size = p[-3] will not return you the size because umm_malloc does not store size, but only next free block.